### PR TITLE
docs: Update install requirements

### PR DESCRIPTION
**Pull request by tara-det-ai**
This is a draft PR to update content related to installation requirements.

changes merged into #7763 

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: docs/setup-cluster/deploy-cluster/firewall-rules.rst [hunk 1]
```diff
@@ -22,6 +25,8 @@ The firewall rules must satisfy the following network access requirements for th
  Agents
 ********
 
+NEEDS UPDATES
+
```
Comment (NicholasBlaskey) [inline docs/setup-cluster/deploy-cluster/firewall-rules.rst:29]: I can't confirm or deny port 1750 @rb-determined-ai do you know?

```
   -  Inbound and outbound TCP ports 1734 and 1750 are used for synchronization between trial
      containers.
```

File: docs/setup-cluster/deploy-cluster/firewall-rules.rst [hunk 2]
```diff
@@ -22,6 +25,8 @@ The firewall rules must satisfy the following network access requirements for th
  Agents
 ********
 
+NEEDS UPDATES
```
Comment (NicholasBlaskey) [inline docs/setup-cluster/deploy-cluster/firewall-rules.rst:28]: The only thing that I think that might be missing is for generic proxies.

Do you know if we have to update any port requirements for that @ioga ?

File: docs/setup-cluster/deploy-cluster/firewall-rules.rst [hunk 3]
```diff
@@ -6,12 +6,15 @@
  Firewall Rules
 ################
 
-The firewall rules must satisfy the following network access requirements for the master and agents.
+DRAFT The firewall rules must satisfy the following network access requirements for the master and
```
Comment (tara-det-ai) [inline docs/setup-cluster/deploy-cluster/firewall-rules.rst:9]: https://hpe-aiatscale.slack.com/archives/CRYLQP7MW/p1696279343216229
* Compute nodes must be able to connect to master node on master's configured port.
* Compute nodes must be able to connect to each other on any port.
* Master node must be able to connect to compute nodes on any port.
* Compute nodes must be able to reach the docker image repository, or have the
  relevant images pre-downloaded.
* Compute nodes must have access to the desired checkpoint storage.
* Master node must also have access to desired checkpoint storage.
  * (this isn't technically required, but it's coming soon)
* Optionally, client nodes may have access to checkpoint storage for higher
  performance checkpoint access.
* Master node must have access to postgres
* Compute nodes must have access to network resources required by user tasks.
  Frequently this includes installing packages from pypi, for instance.
* Client machines must have access to the master node on master's configured port.

File: docs/setup-cluster/deploy-cluster/installation-requirements-temp.rst [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+******************
+
+ADD CONTENT EDITORIAL Likewise this content is meant to provide prerequisites the admin needs to
+fulfill before attempting to set up SSO.
+
+*********************************
+ Non-Root Container Requirements
+*********************************
+
+Might be docs for this
```
Comment (tara-det-ai) [inline docs/setup-cluster/deploy-cluster/installation-requirements-temp.rst:35]: https://docs.determined.ai/latest/setup-cluster/users.html#run-tasks-as-specific-agent-users

**Comment by tara-det-ai** [comment]
changes merged into https://github.com/determined-ai/determined/pull/7763

**Pull request closed without merging**